modules/health_quest/app/docs/health_quest/v0/health_quest.yaml

```
@@ -0,0 +1,251 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Health Quest
+  description: |
+    ## The API for the Healthcare Clipboard 
+
+    These endpoints include the Healthcare Experience Questionaire 
+    Responses and clipboard related endpoints into the MAP PGD and scehduled appointments
```

```suggestion
    Responses and clipboard related endpoints into the MAP PGD and scheduled appointments
```

What is `MAP PGD` ? Please define initialisms on first invocation